packages/discord.js/src/structures/Attachment.js

```diff
@@ -99,6 +99,26 @@ class Attachment {
      * @type {boolean}
      */
     this.ephemeral = data.ephemeral ?? false;
+
+    if ('duration_secs' in data) {
+      /**
+       * The duration of this attachment in seconds (if an audio)
+       * @type {?number}
+       */
+      this.duration = data.duration_secs;
+    } else {
+      this.duration ??= null;
+    }
+
+    if ('waveform' in data) {
+      /**
+       * The base64 encoded bytearray representing a sampled waveform (if an audio)
```

Comment:
```suggestion
       * The base64 encoded bytearray representing a sampled waveform
       * <info>This will only be available if the attachment is an audio file</info>
```

Reply:
There are other properties that use this format in this file. Feel free to update those be updated on a follow-up pr